pages/company/components/Story.tsx

```diff
@@ -0,0 +1,83 @@
+/* eslint-disable react/no-unescaped-entities */
+import { Typography } from "@mui/material";
+import { Box } from "@mui/system";
+import { FC } from "react";
+import styles from "../company.module.scss";
+import { story4 } from "assets/png";
+import Image from "next/image";
+
+const story = [
+	{ title: "	✓ Founded in 2014" },
+	{ title: "	✓ Community of 125K+" },
+	{ title: "	✓ $33M in funding" },
+	{ title: "	✓ 500+ global meetups" },
+];
+export const Story: FC = () => {
+	return (
+		<div className={styles.story}>
+			<Typography
+				variant="h4"
+				sx={theme => ({
+					[theme.breakpoints.down("sm")]: {
+						textAlign: "center",
+					},
+				})}
+			>
+				<Typography variant="h5" mb={3}>
+					OUR STORY
+				</Typography>
+				We’re going to become <br /> partners for long run.
+			</Typography>
+			<Box>
+				<Typography maxWidth={559} textAlign="center">
+					Lorem ipsum dolor sit amet consectetur, adipisicing elit. Et odit
+					quibusdam voluptates dolores pariatur porro ipsum nisi illo,
+					cupiditate iste, asperiores totam exercitationem?{" "}
+				</Typography>{" "}
+				<br />
+				<Box
+					className={styles.cards}
+					sx={theme => ({
+						[theme.breakpoints.down("sm")]: {
+							gridTemplateAreas: '"a" "b" "c" "d" !important',
+							width: "100%",
+						},
```

Comment:
I think this sx is unnecessary. Its copied from other .tsx

@Asmi-1 You did the same mistake na. tell pratham all the required edits (everything that I told you to do in your PR => you may show him your final component).